editoast/migrations/2023-11-24-162712_add_errors_hash/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE infra_layer_error ADD info_hash varchar(40) NOT NULL DEFAULT 'undefined';
+UPDATE infra_layer_error SET info_hash = id::text;
```

Comment:
I didn't want to do this for multiple reasons:
- It will be very difficult to reproduce the same hash that editoast will produce
- The first edition will be slower cause all errors will be removed and inserted with the correct hash. But it's no worse than what we already have. 
- We can think of it as a poorly initialized cache

For me, it was an acceptable compromise to avoid bothering with migration.